cmd/ctr/commands/run/run.go

```diff
@@ -123,6 +123,10 @@ var Command = cli.Command{
 			Name:  "cni",
 			Usage: "Enable cni networking for the container",
 		},
+		cli.StringFlag{
+			Name:  "sandbox",
+			Usage: "Run the container in the given sandbox",
+		},
```

Comment:
I think we can put this flag in `commands.ContainerFlags`, so that ctr container create can also support.